src/rtrc.js

```diff
@@ -623,6 +627,82 @@
 		$('#krRTRC_loader').hide();
 	}
 
+	function applyOresAnnotations($feedContent, callback) {
+		var revids;
+
+		// Find all revids names inside the feed
+		revids = [];
+		$feedContent.filter('.mw-rtrc-item').each(function () {
+			var revid = $(this).attr('data-diff');
+			if (revid) {
+				revids.push(revid);
+			}
+		});
+
+		if (!revids.length) {
+			callback();
```

Comment:
To make this code more robust, use `.always(callback);` at the end of this chain instead of calling it from `.fail()` and twice in `.done()` manually.